examples/guestbook-go/README.md

```diff
@@ -189,6 +184,13 @@ For details about limiting traffic to specific sources, see the [GCE firewall do
 
 ### Step Seven: Cleanup
 
+You should delete the service which will remove any associated resources that were created e.g. load balancers, forwarding rules and target pools.
+```shell
+$ cluster/kubectl.sh delete service guestbook
```

Comment:
Nit: I think you can do a delete -f on the guestbook directory to cleanup all the resources, including the service.